src/lib/stores/drips/utils/__test__/mockObjects.ts

```diff
@@ -0,0 +1,21 @@
+import type { DripHistoryEvent } from '$lib/stores/workstreams';
+import { Currency, type Money } from '$lib/stores/workstreams/types';
+
+export const mockMoney = (amount: bigint | number | string): Money => ({
+  wei: BigInt(amount),
+  currency: Currency.DAI
+});
+
+export const mockDripHistoryEvent = ({
+  balance,
+  amtPerSec,
+  seconds
```

Comment:
Reading the test I got confused as to what this is, wonder if the variable shouldn't be called `timestampSeconds`. I initially thought it's the duration of the event.